SwiftWisdomTests/Rx/Rx+ExtensionsTest.swift

```diff
@@ -9,27 +9,27 @@ class OperatorTests: XCTestCase {
     func testVariableBinding() {
         let disposeBag = DisposeBag()
         let label = UILabel()
-        let variable = Variable<String?>(nil)
+        let variable = BehaviorRelay<String?>(value: nil)
```

Comment:
Tiny thing, but I might consider renaming the bindings in these tests to `relay` or `behaviorRelay` instead of `variable`.

Reply:
Not so tiny - renamed and removed duplicated tests!